vavr-modules/vavr-2/pom.xml

```diff
@@ -19,6 +19,49 @@
             <artifactId>vavr-test</artifactId>
             <version>${vavr.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-data-jpa</artifactId>
+            <version>${spring-boot.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.h2database</groupId>
+            <artifactId>h2</artifactId>
+            <version>${h2.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-test</artifactId>
+            <version>${spring-boot.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
```

Comment:
Are logback dependencies added from you, and not moved from vavr-modules/vavr?

Reply:
Yes, mistakenly added by myself. Updated pr.